core/storage/opportunity/gae_models.py

```diff
@@ -93,7 +93,7 @@ def get_all_translation_opportunities(
             page_size: int,
             urlsafe_start_cursor: Optional[str],
             language_code: str,
-            topic_name: str
+            topic_name: Optional[str]
```

Comment:
Why is this optional?

Reply:
According to docstring, it can accept None:
https://github.com/oppia/oppia/blob/22e557ebd3d5394aacd9c30c45698fb30d8e5a6a/core/storage/opportunity/gae_models.py#L111
Also, in opportunity_service we are providing None:
https://github.com/oppia/oppia/blob/22e557ebd3d5394aacd9c30c45698fb30d8e5a6a/core/domain/opportunity_services.py#L447